### ISSUE TITLE
Cinnamenu@json Alt-Tab does not close the menu

### ISSUE THREAD

**Issue by mahajanudit** [linuxmint/cinnamon-spices-applets#3036]
```
 * Applet version/Build date (to be found in "About.." or in "metadata.json") 4.1.5
 * Cinnamon version (cinnamon --version) Cinnamon 4.4.8
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) Mint
 * Graphics hardware *and* driver used Intel
 * Snippet or attachment of ~/.cinnamon/glass.log
 * Snippet or attachment of ~/.xsession-errors

 ```

**Notify author of applet**
@

**Issue**
Pressing Alt-tab when menu is open, does not close the menu and open app-switcher.
It seems like the keyboard shortcut is just ignored or absorbed. App-switcher Alt-tab should be allowed to close the menu immediately and open the switcher. Instead there is an additional step to close the menu first and switch.

**Steps to reproduce**
Open menu, press alt-tab

**Expected behaviour**
Close the menu and open app-switcher

**Other information**

**Comment by fredcw** [comment]
@mahajanudit Well spotted. I'll take a look at it.

**Comment by fredcw** [comment]
@mahajanudit I don't know how to fix this. The best I can do is close the menu when Alt-Tab is pressed which means you have to press Alt-Tab twice to get the app-switcher. This will be included in the next cinnamenu update in a few days. But this issue seems to be present in all applets and cinnamon menus, even nemo.

**Comment by mahajanudit** [comment]
I see, is there a way you can forward the key press to OS instead of absorbing it? I don't know actually how things work, but if it's possible to either invoke the OS shortcut or forward it to OS, then it is doable. But if it turns out to be too much of an issue, I would say let it be. This is how windows handles it: ![Windows Screenshot](https://drive.google.com/file/d/1zKRfSLydtm-rU-eM48aaowTRMRSwxFbn/preview) 
It seems like it just forwards the event to OS and then on blur closes the menu.

**Comment by fredcw** [comment]
@mahajanudit Yes, what you say sounds like the way to do it and I expect that it's fairly easy to do but I can't find the documentation for how to do it. I took over this applet from someone else and I'm not too familiar with the cinnamon API.
